finalPrep/modZeroHero.js

```diff
@@ -2,52 +2,125 @@
+var catchPhrase = `I am ${specialAbility}!`;
+
+console.log(greeting);
+console.log(catchPhrase);
+
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 75;
+var energy = 250;
+
+
+console.log(power);
+console.log(energy);
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = (power * 500);
+var fullEnergy = (energy + 150);
+
+console.log(fullPower);
+console.log(fullEnergy);
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = false;
+var identityConcealed = true;
+
+console.log(isHuman);
+console.log(identityConcealed);
+
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ['dogs', 'the vet', 'mean rats'];
+var sidekicks = ['sister kitten', 'personal human', 'the mailman'];
+
+console.log (archEnemies)
+console.log(sidekicks)
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(archEnemies[2]);
+
 // Write some code to add a new archEnemy to the archEnemies array
 
+var archEnemy = archEnemies.push('dogs');
+
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log(archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+var lessSidekicks = sidekicks.shift();
+
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+var newSidekick = sidekicks.push('bestie cat');
+
+console.log (sidekicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+var dangerLevel = 50;
+var saveTheDay = "I'll save your tails and whiskers too!";
+var badExcuse = "I think I got a bad bowl of cat food this morning...";
+
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+function assessSituation(dangerLevel) {
+if (dangerLevel > 50) {
+    console.log(badExcuse);
+} else if (dangerLevel >= 10 && dangerLevel <= 50) {
```

Comment:
Remember to have an `else` statement at the end that captures anything else! You can use that to avoid some complicated logic- consider having the `saveTheDay` message being your else statement.